resources/README.md

```diff
@@ -67,13 +67,14 @@ Mindee is an API first platform that helps developers automate applications' wor
 - [GitHub](https://github.com/mindee)
 - [LinkedIn](https://www.linkedin.com/company/mindee/)
 - [Live Tests](https://mindee.com/live-test)
-- [Newsletter](https://cdn.forms-content.sg-form.com/13b540c7-2c28-11ec-a526-325185d2b4f9)
+- [Newsletter](https://newsletter.mindee.com)
 - [SDK Node.js](https://github.com/mindee/mindee-api-nodejs)
 - [SDK Python](https://github.com/mindee/mindee-api-python)
-- [Slack Community](https://join.slack.com/t/mindee-community/shared_invite/zt-uzgmljfl-MotFVfH~IdEZxjp~0zldww)
+- [Slack Community](https://slack.mindee.com)
 - [Twitter](https://twitter.com/mindeeAPI)
 - [Website](https://mindee.com)
 - [YouTube](https://www.youtube.com/channel/UCXcb0H4P81RqvvvFfWdszoA) (video tutorials)
+Add COmputer VVision
 
```

Comment:
There is minor typo here

Reply:
It shouldn't be there, thanks

Reply:
Oh actually, not sure why this README is the old one!